radio/src/targets/9x/lcd_driver.cpp

```diff
@@ -145,6 +192,41 @@ uint8_t lcdRefresh_ST7920(uint8_t full)
 void lcdRefresh()
 {
 #endif
+#if defined(LCD_KS108)
+static uint8_t change = 0; // toggle left or right lcd writing

```

Comment:
Does it mean you always call 2 times lcdRefresh? Why not introduce such a function:

```
void lcdRefresh()
{
  lcdRefreshLeft();
  lcdRefreshRight();
}
```